great_expectations/data_context/data_context/abstract_data_context.py

```diff
@@ -3917,27 +3917,31 @@ def get_available_data_asset_names(
             ):  # Iterate over both together
                 for idx, datasource_name in enumerate(datasource_names):
                     datasource = self.get_datasource(datasource_name)
-                    assert not isinstance(
-                        datasource, FluentDatasource
-                    ), 'Method "get_available_data_asset_names" not implemented for FluentDatasource'
-                    data_asset_names[
-                        datasource_name
-                    ] = datasource.get_available_data_asset_names(
-                        batch_kwargs_generator_names[idx]
-                    )
+                    if isinstance(datasource, FluentDatasource):
+                        data_asset_names[datasource_name] = list(
+                            datasource.get_asset_names()
+                        )
```

Comment:
Nit - probably not super necessary but if we're converting from a set to a list, can we use `sorted()`? It would be nice to get consistent ordering here since we're not guaranteed that in a set.

Same throughout